### ISSUE TITLE
Create `nasa-openscapes-workshops` github org

### ISSUE THREAD

**Issue by jules32** [NASA-Openscapes/workshop-planning#58]

**Comment by stefaniebutland** [comment]
successfully used this Feb 13, 2024 for SWOT workshop at AGU Chapman (@cassienickles & @ScienceCat18 ). Added participants GitHub usernames to https://github.com/orgs/nasa-openscapes-workshops/teams/workshopaccess-2i2c

**Comment by ScienceCat18** [comment]
Yes, this worked beautifully. We get several comments after the workshop of how pleasantly surprised participants were on how smooth the hands-on onboarding and using the cloud hub was (vs as it is often the case people traditionally get stuck using a hands-on system during workshops). Thank you 2i2c, Openscapes and NASA DAAC mentors for building an amazingly efficient and helpful end-to-end system for the NASA DAACs and their data users. 🙌

**Comment by stefaniebutland** [comment]
And thank you @yuvipanda!

**Comment by yuvipanda** [comment]
Yay, this made my day. So glad to hear, @ScienceCat18!

**Comment by jules32** [comment]
Keeping this open so to document this in the Cookbook

**Comment by stefaniebutland** [comment]
Starter documentation: https://nasa-openscapes.github.io/earthdata-cloud-cookbook/leading-workshops/add-folks-to-2i2c-github-teams.html, but process has evolved with creating one Team per workshop, and the possibility of avoiding GitHub login for some workshops (I think)
